pkg/db/migration.go

```diff
@@ -160,8 +161,8 @@ func closeMigrate(m *migrate.Migrate) {
 	}
 }
 
-func MigrateUp(p params.Database) error {
-	m, err := getMigrate(p)
+func MigrateUp(dbParams params.Database, blockParams blockparams.AdapterConfig) error {
```

Comment:
Seems redundant to me...

Reply:
Passing the blockParams to all the Migrate functions

Reply:
See how it's being used, we need it for the blockstore migration